crates/bevy_dev_tools/src/lib.rs

```diff
@@ -35,10 +38,12 @@ pub mod ci_testing;
 pub struct DevToolsPlugin;
 
 impl Plugin for DevToolsPlugin {
-    fn build(&self, _app: &mut App) {
+    fn build(&self, app: &mut App) {
+        app.add_plugins(AabbGizmoPlugin);
```

Comment:
I agree. We have to create this abstraction soon, though, the amount of created features can be very high if we don't handle with care.

Reply:
I think we can do `DevToolsPlugin` into a `PluginGroup`, to make it easy disable and enable things, but I'm not sure how this will play with `DefaultPlugins` though

Reply:
I've thought about it too, but wouldn't it then be impossible to change dev tools during runtime? I also thought about making a config as a resource but there might be a better way

Reply:
I don't think there is a way at disabling this at runtime right now, at least not in the way we are currently doing 
we add plugins to the app after all, is there a way to currently disabling a plugin, adding a plugin, or even adding systems at runtime right now?  If we want, we can at least making it into a group now and mitigate the effects I guess, at least while we don't have the runtime tool manager

Reply:
Yes we can't add/remove plugins at runtime (see #11083), but *maybe* we could achieve something without plugins? I think we should create an issue regarding this to decide which direction we want to go for now